proto/stream_plan.proto

```diff
@@ -110,6 +110,14 @@ message DropSubscriptionsMutation {
   repeated SubscriptionUpstreamInfo info = 1;
 }
 
+message ConnectorPropsInfo {
+  map<string, string> connector_props_info = 1;
+}
+
+message ConnectorPropsChangeMutation {
+  map<uint32, ConnectorPropsInfo> connector_props_infos = 1;
+}
```

Comment:
shall we move the `ConnectorPropsInfo` to the inside of `ConnectorPropsChangeMutation`? Like

```
message ConnectorPropsChangeMutation {
  message ConnectorPropsInfo {
    map<string, string> connector_props_info = 1;
  }

  map<uint32, ConnectorPropsInfo> connector_props_infos = 1;
}
```

because the type is relatively simple and won't be reused elsewhere.

Reply:
& please add some comments about the concept mapping